### PR TITLE
ReadFileInBinaryChunkedMode

### PR DESCRIPTION
**Pull request by Ajanthy**
Siddhi-map-binaryPassThrough mapper is used.

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: modules/samples/artifacts/ReadFileInBinaryChunkedMode/ReadFileInBinaryChunkedMode.siddhi [hunk 1]
```diff
@@ -0,0 +1,60 @@
+@App:name('ReadFileInBinaryChunkedMode')
+
```
Comment (lasanthaS) [inline modules/samples/artifacts/ReadFileInBinaryChunkedMode/ReadFileInBinaryChunkedMode.siddhi:2]: Remove this new line

File: modules/samples/artifacts/ReadFileInBinaryChunkedMode/ReadFileInBinaryChunkedMode.siddhi [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+@App:description('Read the file as binary chunks')
```
Comment (lasanthaS) [inline modules/samples/artifacts/ReadFileInBinaryChunkedMode/ReadFileInBinaryChunkedMode.siddhi:3]: Make this as "Read a binary file as chunks"

File: modules/samples/artifacts/ReadFileInBinaryChunkedMode/ReadFileInBinaryChunkedMode.siddhi [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+/*
+Purpose:
+    This application demonstrates how to configure WSO2 Streaming Integrator Tooling to read the file and send events
```
Comment (lasanthaS) [inline modules/samples/artifacts/ReadFileInBinaryChunkedMode/ReadFileInBinaryChunkedMode.siddhi:7]: Rephrase as: "This application demonstrates how to configure WSO2 Streaming Integrator Tooling to read a binary file in chunks and send via WebSocket transport."

File: modules/samples/artifacts/ReadFileInBinaryChunkedMode/ReadFileInBinaryChunkedMode.siddhi (hunk 1 above)
Comment (Ajanthy) [reply modules/samples/artifacts/ReadFileInBinaryChunkedMode/ReadFileInBinaryChunkedMode.siddhi]: fixed in https://github.com/wso2/streaming-integrator-tooling/pull/87/commits/3b0b69ece28668ca58eaee3041d4cb7636494296

File: modules/samples/artifacts/ReadFileInBinaryChunkedMode/ReadFileInBinaryChunkedMode.siddhi (hunk 2 above)
Comment (Ajanthy) [reply modules/samples/artifacts/ReadFileInBinaryChunkedMode/ReadFileInBinaryChunkedMode.siddhi]: fixed in https://github.com/wso2/streaming-integrator-tooling/pull/87/commits/3b0b69ece28668ca58eaee3041d4cb7636494296

File: modules/samples/artifacts/ReadFileInBinaryChunkedMode/ReadFileInBinaryChunkedMode.siddhi (hunk 3 above)
Comment (Ajanthy) [reply modules/samples/artifacts/ReadFileInBinaryChunkedMode/ReadFileInBinaryChunkedMode.siddhi]: fixed in https://github.com/wso2/streaming-integrator-tooling/pull/87/commits/3b0b69ece28668ca58eaee3041d4cb7636494296

**Pull request merged**